tests/rados/test_bluestoretool_workflows.py

```diff
@@ -32,155 +37,336 @@ def run(ceph_cluster, **kw):
         1 -> Fail, 0 -> Pass
     *** Currently, covers commands/workflows valid only for collocated OSDs
     Commands reserved for future coverage with non-collocated OSDs:
-    - ceph-bluestore-tool reshard --path osd path --sharding new sharding [ --sharding-ctrl control string ]
-    - ceph-bluestore-tool bluefs-bdev-new-wal --path osd path --dev-target new-device
-    - ceph-bluestore-tool bluefs-bdev-new-db --path osd path --dev-target new-device
-    - ceph-bluestore-tool bluefs-bdev-migrate --path osd path --dev-target new-device --devs-source device1
     - ceph-bluestore-tool restore_cfb --path osd path
+    - ceph-bluestore-tool bluefs-bdev-migrate --path osd path --dev-target new-device --devs-source device1
     """
     log.info(run.__doc__)
     config = kw["config"]
     rhbuild = config.get("rhbuild")
     cephadm = CephAdmin(cluster=ceph_cluster, **config)
     rados_obj = RadosOrchestrator(node=cephadm)
     bluestore_obj = BluestoreToolWorkflows(node=cephadm)
+    client = ceph_cluster.get_nodes(role="client")[0]
 
     out, _ = cephadm.shell(args=["ceph osd ls"])
     osd_list = out.strip().split("\n")
 
     try:
-        # Execute ceph-bluestore-tool --help
-        osd_id = random.choice(osd_list)
-        log.info(
-            f"\n --------------------"
-            f"\n Running cbt help for OSD {osd_id}"
-            f"\n --------------------"
-        )
-        out = bluestore_obj.help(osd_id=osd_id)
-        log.info(out)
-
-        # Execute ceph-bluestore-tool fsck --path <osd_path>
-        osd_id = random.choice(osd_list)
-        log.info(
-            f"\n --------------------"
-            f"\n Running consistency check for OSD {osd_id}"
-            f"\n --------------------"
-        )
-        out = bluestore_obj.run_consistency_check(osd_id=osd_id)
-        log.info(out)
-        assert "success" in out
-
-        # Execute ceph-bluestore-tool fsck --deep --path <osd_path>
-        osd_id = random.choice(osd_list)
-        log.info(
-            f"\n --------------------"
-            f"\n Running deep consistency check for OSD {osd_id}"
-            f"\n --------------------"
-        )
-        out = bluestore_obj.run_consistency_check(osd_id=osd_id, deep=True)
-        log.info(out)
-        assert "success" in out
-
-        if rhbuild.split(".")[0] >= "6":
-            # Execute ceph-bluestore-tool qfsck --path <osd_path>
+        if config.get("non-collocated"):
+            # Execute ceph-bluestore-tool --help
```

Comment:
Please add a debug statement stating that non-collocated scenarios are started here.